schemas/2015-08-01-preview/Microsoft.Portal.json

```diff
@@ -196,7 +206,11 @@
           "oneOf": [
             {
               "type": "object",
-              "additionalProperties": {}
+              "additionalProperties": {
+                "type": "object",
+                "properties": {}
+              },
+              "properties": {}
```

Comment:
is it expected that we want both "properties" and "additionalProperties"?

Reply:
Yeah, both are valid. It's just a little more explicit.